src/AwesomeCMSCore/AwesomeCMSCore/Migrations/20180421100545_1.cs

```diff
@@ -5,17 +5,28 @@
 
 namespace AwesomeCMSCore.Migrations
 {
-    public partial class InitDB_20181204_145606 : Migration
+    public partial class _1 : Migration
```

Comment:
Did you use migrate.bat file to create migration ?

Reply:
No. After that I will use migrate.bat